x/concentrated-liquidity/model/pool.go

```diff
@@ -234,6 +234,7 @@ func (p *Pool) ApplySwap(newLiquidity sdk.Dec, newCurrentTick sdk.Int, newCurren
 	return nil
 }
 
+// TODO: finish this function
```

Comment:
Good catch. Let's make an issue to track this, please

Reply:
just did https://github.com/osmosis-labs/osmosis/issues/4110